alphalens/utils.py

```diff
@@ -292,28 +295,31 @@ def common_start_returns(factor, prices, before, after,
         ending_index = min(day_zero_index + after + 1, 
                            len(returns.index))
 
-        series = returns.iloc[starting_index:ending_index, :]
+        equities_slice = set(equities)
+        if demean is not None:
+            demean_equities = demean.loc[timestamp].index.get_level_values('asset')
+            equities_slice |= set(demean_equities)
+
+        series = returns.ix[starting_index:ending_index, equities_slice]
```

Comment:
Oh yes, pandas 0.20.0 deprecates it. I usually prefer .loc and .iloc but this time I really needed to use .ix because it  supports mixed integer and label based access.The alternate version with an .iloc followed by a .loc is slower. I will change the code anyway. sorry about that.